brickftp/brickftp.go

```diff
@@ -64,9 +64,8 @@ func (c Client) Move(oldName string, newName string) error {
 	if err != nil {
 		return err
 	}
-	const successStatus = http.StatusCreated
-	if res.StatusCode != successStatus {
-		return c.err("Move", "expected status %d, got: %d filename: %s", successStatus, res.StatusCode, oldName)
+	if (res.StatusCode < 200) && (res.StatusCode > 300) {
```

Comment:
Not sure if it matters but some 200 statuses might be an indication of an issue https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_Success